configure.ac

```diff
@@ -191,6 +191,11 @@ CXXFLAGS="$orig_cxxflags"
 AC_PROG_CC_C99
 dnl NB: see C11 below
 
+dnl Some special handling for gcc later on
+if test "$CC" = "gcc"; then
+    cc_is_gcc="yes"
```

Comment:
This is not how to test for gcc; it could well be something like `x86_64-linux-gnu-gcc`